QuizWhois.Api/tests.analyzers.ruleset

```diff
@@ -0,0 +1,108 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is not it better to move the file into the solution root and just reference from each project? So we can keep it consistent and avoid copy-pasting